File: src/main/java/ru/trukhmanov/twochairsbackend/controller/HelloController.java
package ru.trukhmanov.twochairsbackend.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;
import ru.trukhmanov.twochairsbackend.util.CurrentUser;

@RestController()
public class HelloController{

    @GetMapping("/hello")
    public String getHello(){
        return "Hello " + CurrentUser.id();
    }
}

File: src/main/kotlin/ru/trukhmanov/twochairsbackend/controller/HelloController.kt
package ru.trukhmanov.twochairsbackend.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController
import ru.trukhmanov.twochairsbackend.util.CurrentUser

@RestController
class HelloController {

    @GetMapping("/hello")
    fun getHello(): String {
        return "Hello ${CurrentUser.id()}"
    }
}
